_maps/configs/syndicate_litieguai.json

```diff
@@ -1,33 +1,34 @@
 {
 	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
 	"map_name": "Li Tieguai-class Rescue Ship",
+	"prefix": "SSV",
 	"map_short_name": "Li Tieguai-class",
 	"description": "A small, nimble, and exceptionally well-built medical response vessel, the Li Tieguai is a recent addition to Cybersun’s fleet, forming a critical component of their Frontier stabilization program. Li Tieguais come equipped with high-end medical equipment, including a selection of Cybersun augments and prosthetics, as well as weaponry and armor sufficient to protect its personnel in the often-dangerous Frontier sectors, so that they can offer premium healthcare (at premium prices) in even the most dangerous of scenarios.",
 	"tags": [
 		"Medical"
 	],
-	"map_path": "_maps/shuttles/independent/independent_litieguai.dmm",
+	"map_path": "_maps/shuttles/syndicate/syndicate_litieguai.dmm",
 	"namelists": [
 		"SPACE",
 		"BEASTS",
 		"NATURAL_AGGRESSIVE"
 	],
 	"job_slots": {
 		"Captain": {
-			"outfit": "/datum/outfit/job/captain",
+			"outfit": "/datum/outfit/job/syndicate/captain/cybersun",
 			"slots": 1
 		},
-		"Chief Medical Officer": {
-			"outfit": "/datum/outfit/job/cmo/medicaldirector",
+		"Medical Director": {
+			"outfit": "/datum/outfit/job/syndicate/cmo",
 			"officer": true,
 			"slots": 1
 		},
 		"Trauma Team Technician": {
-			"outfit": "/datum/outfit/job/paramedic/traumateam",
+			"outfit": "/datum/outfit/job/syndicate/doctor/cybersun",
```

Comment:
This outfit inherits the normal white lab coat from the base MD outfit. You may wish to NULL that out (please. don't use the rain coat.)